index.js

```diff
@@ -30181,6 +30181,8 @@ function BeeSwarmSimulator(DATA){
                         e.currentVal=player.stats[e.triggerType]-e.startVal
                         e.amount.textContent=e.currentVal
                         
+                        // CHQ: I believe the following lines of code can be edited so that the current val is

```

Comment:
At this point I intend to open an issue surrounding changing this label. The issue will be for future resolution.